integration_tests/test_domestic_migration.py

```diff
@@ -314,28 +314,23 @@ def test_addresses_during_moves(
             continue
 
         total_num_moved += mask_moved_units.sum()
-        # Check that the number of moved units that have the same details is very low
```

Comment:
This test is useless because `address_cols` includes address_id but address_id is what was used to define `mask_moved_units`, i.e. it is always 0 < 0.001.